docs/nimble-spright-requirements-adr.md

```diff
@@ -0,0 +1,79 @@
+
+## Decision
+
+- Components with potential for reuse are strongly encouraged to go in Nimble.
+    - In some cases (mentioned in the preceeding section), the component may be added as "incubating". It should not be assumed that doing so will significantly shorten development time.
+- The [Spright repo](https://github.com/ni/spright) will be an alternative to Nimble. Ownership and review of Spright components falls to the contributing developers. It is up to those owners to define and enforce appropriate standards, but they should conform to a common set of Spright-wide minimum requirements (see table below).
+- Product- or service-specific components go in Spright, in a suitably scoped package named like `spright-components-stratus`.
+- TODO: When should (strictly Angular) components go in SystemLinkShared?
+
+The following table compares the requirements for developing a component in different libraries: 
+
+|                            | Approved Spec | Unit Tests | SB* Visual Tests | SB* API Docs | SB* Usage Docs | Approved** Visual Design | Approved** UX Design | Angular/Blazor Support | Proper a11y | Minimal Tech Debt | Mobile Support | 
+| ---------------------------------- | :-----------: | :--------: | :--------------: | :----------: | :------------: | :----------------------: | :------------------: | :--------------------: | :---------: | :---------------: | :------------: |
+| `nimble-components`                | 🟢           | 🟢        | 🟢               | 🟢           | 🟢            | 🟢                       | 🟢                  | 🟢                     | 🟢         | 🟢               | 🟡
+| `nimble-components` (incubating)   | 🟢           | 🟢        | 🟢               | 🟢           | 🟡            | 🟡                       | 🟡                  | 🟡                     | 🟡         | 🟡               | 🟡
+| `spright-components-<product>`     | 🟡           | 🟢        | 🟢               | 🟢           | 🟡            | 🟡                       | 🟡                  | 🟡                     | 🟡         | 🟡               | 🟡
+
+🟢 = required\
+🟡 = optional\
+*SB = Storybook\
+**By a UX and/or visual designer
+
+
+## Consequences
+
+<!--
+This section describes the resulting context, after applying the decision. All
+consequences should be listed here, not just the "positive" ones. A particular
+decision may have positive, negative, and neutral consequences, but all of them
+affect the team and project in the future.
+-->
+- Spright's infrastructure increases the ease and consistency of UI component development (compared to creating them in an application).
```

Comment:
I guess I was thinking it would be easier to create components in Spright, since it has everything already set up (including patterns, utilities, examples to follow, etc), but I wasn't considering that it's not really comparable to direct Angular/Blazor component development. I'm just going to remove this line.